pkgs/development/python-modules/ruamel-yaml-string/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "ruamel-yaml-string";
+  version = "0.1.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "ruamel.yaml.string";
+    hash = "sha256-enrtzAVdRcAE04t1b1hHTr77EGhR9M5WzlhBVwl4Q1A=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  doCheck = false;
```

Comment:
Why do you disable tests?

Reply:
There are not checks. I meant to make it explicit, but I'll delete it.